### PR TITLE
[WiP] Kinto.js 1.0 release annoucement.

### PR DESCRIPTION
**Pull request by n1k0**
r=? @Natim @leplatrem @michielbdejong 

Note: please wait for the actual release before merging ;)

### CODE REVIEW

File: content/2015.10.kinto-js-realeased.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+   :alt: Kinto.js synchronization flow
+
+A case for encryption
+---------------------
+
+Remote data storage is certainly a concerns-raising topic when it comes to security and privacy.
+
+To help addressing these issues, Kinto.js exposes a `Remote Transformers API <http://kintojs.readthedocs.org/en/latest/api/#transformers>`_, which allow processing records before they're sent to the server and after they've been retrieved from it.
+
+This is a perfect hook for supporting end-to-end encryption, which `we already wrote about earlier </en/kinto-encryption-example>`_.
```
Comment (Natim) [inline content/2015.10.kinto-js-realeased.rst]: I think you must use the  `{filename}/en/kinto-encryption-example` url

**Comment by Natim** [comment]
Seems good to me :+1:

File: content/2015.10.kinto-js-realeased.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+-------------------
+
+We've listed Kinto.js current limitations — the ones we're aware of — in a `dedicated section of the documentation <http://kintojs.readthedocs.org/en/latest/limitations/>`_.
+
+Sure, that's a bunch of work remaining, and you're obviously `invited to contribute <http://kintojs.readthedocs.org/en/latest/hacking/>`_ if you want to help!
+
+The future of Kinto.js
+----------------------
+
+Of course 1.0 is a very first step towards a broader goal; foremost, we absolutely want to support bulk updates leveraging IndexedDB transactions, for performance and consistency purposes.
```
Comment (leplatrem) [inline content/2015.10.kinto-js-realeased.rst]: leveraging x3... taking advantage of ?

File: content/2015.10.kinto-js-realeased.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+So stay tuned for next upcoming releases, and feel free to provide feedback on the `project mailing-list <https://mail.mozilla.org/listinfo/kinto>`_ or by `filing an issue on Github <https://github.com/Kinto/kinto.js/issues/new>`_ :)
+
```
Comment (leplatrem) [inline content/2015.10.kinto-js-realeased.rst:80]: A _«getting started»_ paragraph mentioning the boilerplate ?

File: content/2015.10.kinto-js-realeased.rst [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+:lang: en
+:url: kinto-js-1-0-released
+:slug: kinto-js-1-0-released
+:date: 2015-10-27
+:summary: After months of development, we're proud to announce the release of Kinto.js 1.0.0 stable.
+
+Kinto.js_ is an offline-first JavaScript client leveraging the `Kinto API <https://kinto.readthedocs.org/>`_ for data synchronization. In a world where connectivity isn't always guaranteed, local persistence in the browser has become an `identified priority <http://offlinefirst.org/>`_ for Web applications, especially on mobile.
+
+The base principle is simple; save your data locally, assuming there's no network connectivity. Then as soon as it's available, retrieve a remote changes and push your local data to a remote server.
```
Comment (tarekziade) [inline content/2015.10.kinto-js-realeased.rst]: basic principle

Comment (tarekziade) [inline content/2015.10.kinto-js-realeased.rst]: s/retrieve a remote changes/retrieve remote changes/

File: content/2015.10.kinto-js-realeased.rst [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+The diagram flow below describes pretty much accurately what's happening under the hood when ``sync()`` is called:
+
+.. image:: {filename}/images/kintojs-sync-flow.png
+   :alt: Kinto.js synchronization flow
+
+A case for encryption
+---------------------
+
+Remote data storage is certainly a concerns-raising topic when it comes to security and privacy.
```
Comment (tarekziade) [inline content/2015.10.kinto-js-realeased.rst]: s/concerns-raising topic/topic raising concerns/   (i don't think concerns-raising is an adjective in english)

File: content/2015.10.kinto-js-realeased.rst [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+-------------------
+
+We've listed Kinto.js current limitations — the ones we're aware of — in a `dedicated section of the documentation <http://kintojs.readthedocs.org/en/latest/limitations/>`_.
+
+Sure, that's a bunch of work remaining, and you're obviously `invited to contribute <http://kintojs.readthedocs.org/en/latest/hacking/>`_ if you want to help!
+
+The future of Kinto.js
+----------------------
+
+Of course 1.0 is a very first step towards a broader goal; foremost, we absolutely want to support bulk updates taking advantage of IndexedDB transactions, for performance and consistency purposes.
```
Comment (tarekziade) [inline content/2015.10.kinto-js-realeased.rst]: s/taking/by taking/

**Comment by michielbdejong** [comment]
r+ Great write-up! And congrats on the big event. :)

**Comment by n1k0** [comment]
As 1.1 is about to be released, I'll hold on publishing the post as-is; I'll update it after the new release.

File: content/2015.10.kinto-js-realeased.rst [hunk 7]
```diff
@@ -0,0 +1,88 @@
+Kinto.js 1.0 released
+#####################
+
+:lang: en
+:url: kinto-js-1-0-released
+:slug: kinto-js-1-0-released
+:date: 2015-10-27
+:summary: After months of development, we're proud to announce the release of Kinto.js 1.0.0 stable.
+
+Kinto.js_ is an offline-first JavaScript client leveraging the `Kinto API <https://kinto.readthedocs.org/>`_ for data synchronization. In a world where connectivity isn't always guaranteed, local persistence in the browser has become an `identified priority <http://offlinefirst.org/>`_ for Web applications, especially on mobile.
```
Comment (almet) [inline content/2015.10.kinto-js-realeased.rst:10]: consider: "especially on mobile _platforms_"

**Comment by almet** [comment]
+1! Thanks for the article, and big up for the release :-)

**Pull request merged**